components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1736,6 +1737,146 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
             .map((s) => s.url);
     }
 
+    public async getSuggestedRepositories(ctx: TraceContext, organizationId: string): Promise<SuggestedRepository[]> {
+        const user = await this.checkAndBlockUser("getSuggestedRepositories");
+        await this.guardWithFeatureFlag("includeProjectsOnCreateWorkspace", user, organizationId);
```

Comment:
why do we guard by feature flag here? wouldn't it be better to guard on the UI side? There doesn't seem a ton of value in throwing errors for some users when they are not enabled.

Reply:
The UI is guarded too, I can remove it here.